packs/feat-effects/effect-expel-maelstrom-failure.json

```diff
@@ -0,0 +1,68 @@
+{
+    "_id": "lc8ez8O4eN73hNpR",
+    "img": "icons/magic/control/energy-stream-link-spiral-white.webp",
+    "name": "Effect: Expel Maelstrom (Failure)",
```

Comment:
These names do not make it obvious what is happening. To get the crit fail state you apply the fail state then select crit fail? Even just changing the name from crit fail to (Reroll) or (Misfortune) or something like that I think would make it clearer what we're doing.

Wish we had duration alterations for this stuff.

Reply:
I feel you on that. 

You're right, it's unclear. it Would you rather the `Expel Maelstrom (Failure)` effect be renamed to `Expel Maelstrom (Failure or Critical Failure)` and then the one that handles the Roll Twice RE be renamed `Expel Maelstrom (Misfortune)`? I think that'd be a clearer way to do about it.

Reply:
Yeah, with potentially adding these to the issue for condensing later when we get [more magic](http://www.catb.org/jargon/html/magic-story.html)